test/blackbox-tests/test-cases/external-lib-deps/fix-bug.t/lib/dune

```diff
@@ -0,0 +1,3 @@
+(library
+ (name inter_lib)
+ (libraries a________))
```

Comment:
why are tests using names with lots of underscores? would `a` work?

Reply:
to be sure, not having an existent lib.